tests/CMakeLists.txt

```diff
@@ -41,6 +46,28 @@ function(ConfigureTestInternal TEST_NAME)
                            PUBLIC $<$<COMPILE_LANGUAGE:CXX>:-Wno-deprecated-declarations>)
   endif()
 
+  if (CODE_COVERAGE)
+    if(CMAKE_COMPILER_IS_GNUCXX)
+      set(KEEP_DIR ${CMAKE_CURRENT_BINARY_DIR}/tmp)
```

Comment:
```suggestion
```

Reply:
Why delete this?

Reply:
I proposed deleting it as I don't see any usage of the created keep directory. I see you passing `--keep` but not `--keep-dir=${KEEP_DIR}`. Am I missing something?

Reply:
Ah, forgot that I deleted that. Compilation failed the first time I tried --keep-dir, but I think it's because it was a parallel build. Will probably add it back for tidyness.